setup.py

```diff
@@ -221,9 +220,7 @@ def run(self):
     cmdclass=cmdclass,
     # When changing this, also update mypy-requirements.txt.
     install_requires=[
-        "typed_ast >= 1.4.0, < 2; python_version<'3.8'",
```

Comment:
There are several other places where the typed-ast, types-typed-ast, and typing_extensions requirements are specified, which should be kept in sync:

https://github.com/python/mypy/blob/19c5d5f7848a94c810c8d840a42e29ffe3f8906f/build-requirements.txt#L5
https://github.com/python/mypy/blob/19c5d5f7848a94c810c8d840a42e29ffe3f8906f/mypy-requirements.txt#L5
https://github.com/python/mypy/blob/19c5d5f7848a94c810c8d840a42e29ffe3f8906f/pyproject.toml#L12